pomm-project/pomm-bundle/2.0/etc/pomm.yaml

```diff
@@ -0,0 +1,10 @@
+pomm:
+    configuration:
+        my_db1:
+            dsn: '%env(DATABASE_DSN)%'
+#            pomm:default: true
```

Comment:
I saw, I don’t have a strong opinion about that. It’s maybe easier to maintain without comment (each new configuration require a new pull request, errors in comment are silently).